composer/callbacks/health_checker.py

```diff
@@ -3,26 +3,19 @@
 
 """Check GPU Health during training."""
 import logging
+import os
 from collections import deque
 from datetime import datetime
 from typing import List, Optional, Tuple
 
-import torch
-
-try:
-    import pynvml
-except ImportError:
-    pynvml = None
-
-import os
-
 import numpy as np
-from slack_sdk.webhook import WebhookClient
+import torch
 
 from composer.core import Callback, State
 from composer.core.time import Timestamp
 from composer.loggers import Logger
 from composer.utils import dist
+from composer.utils.import_helpers import MissingConditionalImportError
```

Comment:
```suggestion
from composer.utils import dist, MissingConditionalImportError
```
```suggestion
from composer.utils import MissingConditionalImportError
```
nit: import from toplevel